### DISCUSSION TITLE
Upvote from history page

### DISCUSSION THREAD

**Discussion by MrGrigri** [dailydotdev/daily#523] [Feature Requests]
**Update**

I just now figured out that if you click on the article, you can thumbs-up it from there. Would still be nice to have a quick action in the history page.

**Situation**

Recently, I had a new tab open to Daily.dev for quite some time and it hadn't refreshed. I saw an article that I wanted to read. I clicked on it and then closed the Daily.dev tab out of habit. After reading the article, I wanted to thumbs-up the article. When I opened a new Daily.dev tab, the article was gone. I saw that there was a history tab so I clicked on it thinking that I could thumbs-up it from there. To my disappointment, I tried looking everywhere and couldn't find it.

**TL/DR**

Create the ability to thumbs-up (or comment on) an article from the history page. 

**Thanks for all that y'all do!** I really do appreciate this free resource.

**Comment by idoshamun** [comment]
Thanks a lot @MrGrigri!
It's a great idea :)
